pycheribuild/projects/run_qemu.py

```diff
@@ -259,9 +260,30 @@ def gdb_command(main_binary, breakpoint=None, extra_binary=None) -> str:
                 self.info("If you would like to debug a userspace program (e.g. sbin/init):")
                 self.info("\t", coloured(AnsiColour.red, gdb_command(self.rootfs_path / "sbin/init", "main", path_to_kernel)), sep="")
             self.info("Launching QEMU in suspended state...")
-            # TODO: control tmux to do this automatically?
-            # See e.g. https://github.com/tmux-python/libtmux/
-            qemuCommand += ["-gdb", "tcp::1234",  # wait for gdb on localhost:1234
+
+            def start_gdb_in_tmux_pane(command):
+                server = libtmux.Server()
+                if server is None:
+                    raise Exception("Tmux server not found")
+                sessions = server.list_sessions()
+                if len(sessions) != 1:
+                    raise Exception("There should be only one tmux session running")
+                session = server.list_sessions()[0]
+                window = session.attached_window
+                pane = window.attached_pane.split_window(vertical=True, attach=False)
```

Comment:
I would prefer a horizontal split, but I don't really mind.

Reply:
Now it is set based on the window size